products/green_fast_track/models/intermediate/flags/int_flags__zoning.sql

```diff
@@ -2,50 +2,93 @@ WITH pluto AS (
     SELECT * FROM {{ ref('stg__pluto') }}
 ),
 
-zoning_districts_exploded AS (
+district_mappings AS (
+    SELECT * FROM {{ ref('zoning_district_mappings') }}
+),
+
+district_categories AS (
+    SELECT * FROM {{ ref('zoning_district_categories') }}
+),
+
+districts_exploded AS (
     SELECT
         bbl,
-        UNNEST(ARRAY[zonedist1, zonedist2, zonedist3, zonedist4]) AS zd
+        UNNEST(ARRAY[zonedist1, zonedist2, zonedist3, zonedist4]) AS zd,
+        COALESCE(zonedist1, zonedist2, zonedist3, zonedist4) IS null AS zd_all_null
```

Comment:
Should be able to just check zonedist1 - pluto has logic to ensure that you never have zonedist{n+1} is not null if zonedist{n} is null